requirements/extra.txt

```diff
@@ -0,0 +1,15 @@
+# Extra dependencies required for development and tests
+
+-r base.txt
+
+# Pytest and the optional pytest coverage plugin for tests
+pytest
+pytest-cov
+
+# Optional dependencies for extra functionality
+sqlalchemy
+scikit-image
```

Comment:
Also, the Travis build failed while running this PR but I don't think it has anything with these changes. https://travis-ci.org/sunpy/sunpy/jobs/53914412

Don't have enough knowledge to debug it